tests/parametric/test_otel_span_methods.py

```diff
@@ -330,21 +588,797 @@ def test_otel_set_span_status_ok(self, test_agent, test_library):
-                parent.end_span()
-                with test_library.otel_start_span(name="operation", parent_id=parent.span_id) as span:
-                    span.end_span()
+            with test_library.otel_start_span(name="ok_span", span_kind=SK_INTERNAL) as span:
+                span.set_status(OTEL_OK_CODE, "ok_desc")
+                span.set_status(OTEL_ERROR_CODE, "error_desc")
+                span.end_span()
+
+        span = get_span(test_agent)
+        assert span.get("meta").get("error.message") is None
+        assert span.get("name") == "internal"
+        assert span.get("resource") == "ok_span"
+
+    def test_otel_get_span_context(self, test_agent, test_library):
+        """
+            This test verifies retrieving the span context of a span
+            accordingly to the Otel API spec
+            (https://opentelemetry.io/docs/reference/specification/trace/api/#get-context)
+        """
+        with test_library:
+            with test_library.otel_start_span(name="operation") as parent:
+                parent.end_span()
+                with test_library.otel_start_span(name="operation", parent_id=parent.span_id) as span:
+                    span.end_span()
                     context = span.span_context()
                     assert context.get("trace_id") == parent.span_context().get("trace_id")
                     assert context.get("span_id") == "{:016x}".format(span.span_id)
                     assert context.get("trace_flags") == "01"
+
+    @missing_feature(context.library == "golang", reason="Not implemented")
+    @missing_feature(context.library == "java", reason="Not implemented")
+    @missing_feature(context.library == "nodejs", reason="Not implemented")
+    @missing_feature(context.library == "dotnet", reason="Not implemented")
+    @missing_feature(context.library == "python", reason="Not implemented")
+    @missing_feature(context.library == "python_http", reason="Not implemented")
+    def test_otel_span_operation_name_http_server(self, test_agent, test_library):
+        """
+            Tests that the operation name will be set to "http.server.request" when:
+            - Span kind is set to Server
+            - http.request.method is set to something (e.g., GET in this example)
+
+            (https://opentelemetry.io/docs/specs/otel/trace/semantic_conventions/http/)
+        """
+        with test_library:
+            with test_library.otel_start_span("otel_span_name", span_kind=SK_SERVER) as span:
+                span.set_attributes({"http.request.method": "GET"})
+                span.end_span()
+        traces = test_agent.wait_for_num_traces(1)
+        trace = find_trace_by_root(traces, otel_span(name="otel_span_name"))
+        assert len(trace) == 1
+
+        root_span = get_span(test_agent)
+
+        assert root_span["name"] == "http.server.request"
+        assert root_span["resource"] == "otel_span_name"
```

Comment:
```suggestion
        span = get_span(test_agent)
        assert span["name"] == "http.server.request"
        assert span["resource"] == "otel_span_name"
```
nit: just to keep things shorter and simpler, given the number of tests in this file. (Here and throughout)

Reply:
Thanks, done throughout in 74b668ed532bbe54657cc2701cecddf8491d71e9